content/altinn-studio/guides/development/options/functionality/sorting/_index.en.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Sorting
+description: Sorting the options in the list
+weight: 300
+---
+
+Options are usually displayed in the order they are defined in, but it is also possible to sort them alphabetically by
+label. This can be useful to make it easier for the user to find the option they are looking for when the list does not
+need to be in a specific order.
```

Comment:
I fell into quite the rabbit hole when I went to search for some examples where sorting options would break things, but yes, I think in some cases it would be harder for the user to find the correct option when sorted alphabetically. :sweat_smile: Likert may be an example, as the options there should not be sorted in any way.